policyengine_us/parameters/gov/irs/income/exemption/overtime/hce_salary_threshold.yaml

```
@@ -0,0 +1,14 @@
+description: The US classifies as exempt from overtime salaried individuals if they have an annual salary level of or above this value.
+metadata:
+  unit: currency-USD
+  period: year
+  label: highly compensated employee salary threshold
```

Please add to other parameters that are updated with earnings.
```suggestion
  label: highly compensated employee salary threshold
   # Approximation of Secretary of Labor's requirement to update each 3 years
   # "to reflect current earnings data".
  uprating: gov.ssa.nawi
```